src/styles/components/_content-card.scss

```diff
@@ -16,21 +16,21 @@
 
   &-ghost {
     @extend .bsds-card;
-    color: $neutral-100;
+    color: $text-color-ghost;
   }
 
   &-border-light {
     @extend .bsds-card;
     box-sizing: border-box;
-    border: space(xxs) solid $border-color-subtle;
+    border: $border-width-xxs solid $border-color-subtle;
     padding: space(3x);
-    background-color: $neutral-100;
+    background: $neutral-100;
   }
 
   &-border-ghost {
     @extend .bsds-card-ghost;
     @extend .bsds-card-border-light;
-    background-color: rgba($neutral-100, 0.2);
+    background: rgba($neutral-100, 0.2);
```

Comment:
Should this be `$border-color-ghost`?

Reply:
this is for the translucent background, so we can leave it as neutral-100